config/README.md

```diff
@@ -204,7 +204,4 @@ fexec = lithops.FunctionExecutor(rabbitmq_monitor=True)
 |standalone | soft_dismantle_timeout | 300 |no| Time in seconds to stop the VM instance after a job **completed** its execution |
 |standalone | hard_dismantle_timeout | 3600 | no | Time in seconds to stop the VM instance after a job **started** its execution |
 |standalone | exec_mode | consume | no | If set to  **create** standalone backend will automatically create VMs based on the standalone backend|
-|standalone | disable_log_monitoring | False | no | If set to  True pull remote logs will be disabled. This can improve running time|
-|standalone | local_runtime_load | False | no | If set to  True the assumption is that compute backend VM contains required docker image locally. This saves docker pull or load|
-|standalone | use_http | false | no | `true` or `false`. Whether or not use http connections to communicate with the VM instance. If `falase` (default) it communicates with the VM using ssh connections. If true, it uses the encryption_key to encrypt the payload sent trough http.|
```

Comment:
why "use_http", "disable_log_monitoring" were removed?

Reply:
yes, not needed anymore

Reply:
@JosepSampe how it will know if to load runtime from the image or use docker pull?

Reply:
@JosepSampe ahh...i see it now "pull_runtime"

Reply:
yep, I changed the variable name to `pull_runtime` since this variable only activate/deactivate the command `docker pul ...`